### PR TITLE
docs: clean up information about virtual envs

### PR DESCRIPTION
**Pull request by jfcherng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,14 +20,15 @@ Project specific settings can also be set for LSP-pyright (and all LSP plugins):
 
 - From the command palette `Project: Edit Project`
 
-Your project will also require a `pyrightconfig.json` file, usually in the project root.
-
-At a minimum, the file should define where your Python virtualenvs are located and the name of the one to use for your project:
+You may add a `pyrightconfig.json` file to your project root.
+At a minimum, the file should define where your Python `virtualenvs` are located
```
Comment (rchl) [inline README.md:24]: That sentence is also not entirely true. One can have a configuration file without any venv-specific settings. I feel that the Virtual Environment case should have its own section and then this sentence would be relevant there.

Comment (jfcherng) [reply README.md:24]: I guess I would remove this part of changes in this PR and let other people who are familiar with to write that section.

**Comment by jfcherng** [comment]
Feel free to merge this when it's done ;)

File: README.md [hunk 2]
```diff
@@ -18,28 +18,29 @@ Here are some ways to configure the package and the language server.
...
-}
-```
 - Through a `pyrightconfig.json` configuration file (check [settings documentation](https://github.com/microsoft/pyright/blob/master/docs/configuration.md))
 
 ### Virtual environments
 
-If your project needs to run and be validated within an virtual environment, the `pyrightconfig.json` file needs to be
-present at the root of your project.
+If your project needs to run and be validated within a virtual environment, the `pyrightconfig.json` file needs to be
+presented at the root of your project.
```
Comment (rchl) [inline README.md:43]: "present" was correct.

Comment (jfcherng) [reply README.md]: Sorry for my bad English :(

**Pull request merged**